src/deploy/RPM_build/noobaa.spec

```diff
@@ -89,12 +89,14 @@ mv $RPM_BUILD_ROOT/usr/local/noobaa-core/src/deploy/standalone/noobaa_syslog.con
 
 mkdir -p $RPM_BUILD_ROOT/etc/logrotate.d/noobaa
 ln -s /usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.conf $RPM_BUILD_ROOT/etc/logrotate.d/noobaa/logrotate_noobaa.conf
+mv $RPM_BUILD_ROOT/usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.sh $RPM_BUILD_ROOT/etc/logrotate.d/noobaa/logrotate_noobaa.sh
 
 %files
 /usr/local/noobaa-core
 %{_unitdir}/noobaa.service
 /etc/logrotate.d/noobaa/logrotate_noobaa.conf
 /etc/rsyslog.d/noobaa_syslog.conf
+/etc/logrotate.d/noobaa/logrotate_noobaa.sh
```

Comment:
is this a good place to put scripts? I am not so sure. Why not leave it in /usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.sh ?

Reply:
sure we can keep it in `/usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.sh `